amethyst_gltf/src/format/material.rs

```diff
@@ -114,12 +114,7 @@ fn deconstruct_image(data: &TextureData, offset: usize, step: usize) -> TextureD
     use gfx::format::SurfaceType;
     match *data {
         TextureData::Image(ref image_data, ref metadata) => {
-            let metadata = metadata
-                .clone()
-                .with_size(
```

Comment:
Why remove size? You won't know size from the data itself from gltf.

Reply:
Yes, it's in the image data. Let me check one sec.

Reply:
Fixed.